README.md

```diff
@@ -68,10 +52,6 @@ make local
 If a specific namespace is provided only that project will watched. 
 As we reuse `openshift`'s imagestreams for build, we need to access all namespaces.
 
-* Make sure minishift is running and use myproject
-```
-oc project myproject
-```
 * Clean previously created resources
 ```
 make deploy-clean
```

Comment:
@corinnekrych, this target no longer exists because you've removed it with the removal of `make/deploy.mk`.

Reply:
My bad, you've re-added it. Sorry.